addons/survey/views/survey_templates_statistics.xml

```diff
@@ -392,33 +395,36 @@
         </div>
     </template>
 
-    <template id="question_result_choice" name="Question: choice result (simple_choice, multiple_choice)">
-        <ul class="nav nav-tabs d-print-none" role="tablist">
-            <li t-if="question_answered" class="nav-item">
-                <a t-att-href="'#survey_graph_question_%d' % question.id"
-                    t-att-aria-controls="'survey_graph_question_%d' % question.id"
-                    class="nav-link active" data-bs-toggle="tab" role="tab">
-                    <i class="fa fa-bar-chart-o"></i>
-                    <span t-if="question.question_type == 'simple_choice'">Pie Graph</span>
-                    <span t-else="">Graph</span>
-                </a>
-            </li>
-            <li class="nav-item">
-                <a t-att-href="'#survey_data_question_%d' % question.id"
-                    t-att-aria-controls="'survey_data_question_%d' % question.id"
-                    t-attf-class="nav-link #{'active' if not question_answered else ''}"
-                    data-bs-toggle="tab" role="tab">
-                    <i class="fa fa-list-alt me-1"/>Data
-                </a>
-            </li>
-        </ul>
+    <template id="question_result_choice" name="Question: choice result (simple_choice, multiple_choice, scale)">
+        <div class="d-flex justify-content-between">
```

Comment:
Why such changes ? Anyway can't you add classes in ul and avoid lot of nested divs ?